cmd/sdk-server/main.go

```diff
@@ -56,56 +59,33 @@ var (
 )
 
 func main() {
```

Comment:
@enocom I refactored this to look more like the controller main, so I could also do the grpc-gateway proxy.

Would love your eyes on it.